test/integration/test_expid.py

```diff
@@ -110,3 +134,127 @@ def test_copy_minimal(has_min_yaml, autosubmit_exp, autosubmit):
                     description="Pytest experiment")
             assert exc_info.value.code == 7011
             assert "minimal.yml" in str(exc_info.value)
+
+def test_create_expid_default_hpc(autosubmit: Autosubmit) -> None:
+    """Create expid with the default hcp value (no -H flag defined).
+
+    .. code-block:: console
+
+        autosubmit expid -d "test descript"
+
+    :param autosubmit: Autosubmit interface that instantiate with no experiment.
+    :type autosubmit: Autosubmit
+
+    :return: None
+    """
+    # create default expid
+    platform_name = "local"
+    experiment_id = autosubmit.expid(
+        'experiment_id',
+        platform_name,
```

Comment:
As you are passing `local` here, I think that means the docs are inconsistent, as you are instead doing `autosubmit expid -d "experiment_id" -H "locaL"`. Maybe update the docs, but make the arg here `""` and use `"local"` directly in the `assert` statement.